client/socketAddr.js

```diff
@@ -0,0 +1 @@
+window.socketAddr = 'http://localhost:8081';
```

Comment:
Portfile is a serverside file, so it can't easily be served to the frontend. I only intended this bit for testing anyway. We can discuss better solutions for the live version.